LocalSettings.php

```diff
@@ -1948,6 +1953,7 @@
 	),
 	'wmgUseMobileFrontend' => array(
 		'default' => true,
+		'avalicearchiveswiki' => true,
```

Comment:
You can remove this line, MobileFrontend is enabled by default as true, so this does nothing.

Reply:
Right. I hope it's now removed.